src/schemas/user.schema.ts

```diff
@@ -0,0 +1,7 @@
+import * as mongoose from 'mongoose';
+const { Schema } = mongoose;
+export const UserSchema = new Schema({
+  email: { type: String, required: true },
```

Comment:
If this is userSchema, are we not storing user's firstName, lastName, google/fb/linkedin authToken here?? what about resume data which would be parsed from api.
Key missing in schema - token, user_id, slug, resume_info object, resume_url.

Reply:
template_type.